css/style.css

```diff
@@ -0,0 +1,22 @@
+
+
+body{
+    font-family: 'poppins',sans-serif;
+    margin: 5px;
+}
+section{
+justify-content: center;
+text-align: center;
+}
+h1{
+    text-align: center;
+  
+    justify-content: center;
+   
+}
+img{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
+}
```

Comment:
Kindly add linters to your project, link to linters: https://github.com/microverseinc/linters-config. It makes the CSS look much cleaner and easier to navigate 😎 👌